mite_browser/__init__.py

```diff
@@ -1,14 +1,16 @@
 import asyncio
-from bs4 import BeautifulSoup
+import re
 from urllib.parse import urlencode, urljoin
-from re import compile as re_compile, IGNORECASE
+
+from bs4 import BeautifulSoup
+
+import mite_http
 from mite import ensure_fixed_separation
 from mite.exceptions import MiteError
-import mite_http
 
-# TODO: fix
-EMBEDDED_URL_REGEX = re_compile(
-    "\(\s*[\\]?[\"']([^\"':.]*:)?([^\"':.]*\.[^\"':.]*)[\\]?[\"']\s*\)", IGNORECASE  # noqa: W605
+EMBEDDED_URL_REGEX = re.compile(
+    "\\(\\s*[\\]?[\"']([^\"':.]*:)?([^\"':.]*\\.[^\"':.]*)[\\]?[\"']\\s*\\)",
```

Comment:
I guess an r string might help make this slighly more readable? I think this is the same:
`r"""\(\s*[\]?["']([^"':.]*:)?([^"':.]*\.[^"':.]*)[\]?["']\s*\)"""`

Reply:
Good spot, made that change